### PR TITLE
Added wireframe to readme

### PR DESCRIPTION
**Pull request by paulauus**
I have added the wireframe images to the dashboard readme. Lots of thought and nerves has gone into it. Talk to me if you want more info.

Closes issue #9 and #13

### CODE REVIEW

File: dashboard/wireframe_subscribe.png
Comment (mehrdadhalali) [inline dashboard/wireframe_subscribe.png:1]: Put a "include NSFW" tickbox here.

**Review by Leon-JS02 — DISMISSED** [review]
This is snazzy dashboarding.

**Review by MichaelHassan — COMMENTED** [review]
Could we add a page for unsubscribing, or a part of a page.

File: dashboard/wireframe_subscribe.png
Comment (Ruyzambrano) [inline dashboard/wireframe_subscribe.png:1]: Nice, although I might like a background for the Current Subscribers in Each Genre for better contrast. Also consider using units and axis labels for the number of subscribers.

**Review by Leon-JS02 — DISMISSED** [review]
Re-approval, good job applying the suggested changes!

File: dashboard/wireframe_subscribe.png
Comment (mehrdadhalali) [inline dashboard/wireframe_subscribe.png:1]: Where are the final. buttons for subscribe/unsubscribe?

Comment (mehrdadhalali) [inline dashboard/wireframe_subscribe.png:1]: The word "NSFW" on its own is too ominous imo. Could we change it to "Include NSFW" perhaps?

**Review by mehrdadhalali — CHANGES_REQUESTED** [review]
Sorry to be a Mehrdad

File: dashboard/wireframe_subscribe.png
Comment (paulauus) [reply dashboard/wireframe_subscribe.png:1]: :')

Comment (paulauus) [reply dashboard/wireframe_subscribe.png:1]: Done. Please, no more Mehrdading.

Comment (mehrdadhalali) [inline dashboard/wireframe_subscribe.png:1]: Looks good!

**Review by mehrdadhalali — APPROVED** [review]

**Pull request merged**